src/components/Exchange/SwapBox.js

```diff
@@ -313,20 +313,64 @@ export default function SwapBox(props) {
   const toTokenInfo = getTokenInfo(infoTokens, toTokenAddress);
   const toTokenAvailableUsd = toTokenInfo.availableUsd;
 
-  const renderAvailableLongLiquidity = () => {
-    if (!isLong) {
-      return null;
-    }
+  const hasMaxAvailableShort = isShort && toTokenInfo.maxAvailableShort && toTokenInfo.maxAvailableShort.gt(0);
```

Comment:
does every token have a max available long?

Reply:
yeah, I am not even sure what this does tbh, but this is old gmx code. Just going to leave it